Source/ARTTypes.h

```diff
@@ -221,88 +403,129 @@ NSString *generateNonce(void);
 
 @end
 
-#pragma mark - ARTChannelOccupancy
-
+/**
+ * BEGIN CANONICAL PROCESSED DOCSTRING
+ * Contains the metrics of a `ARTRestChannel` or `ARTRealtimeChannel` object.
+ * END CANONICAL PROCESSED DOCSTRING
+ */
 @interface ARTChannelOccupancy : NSObject
 
+/**
+ * BEGIN CANONICAL PROCESSED DOCSTRING
+ * A `ARTChannelMetrics` object.
```

Comment:
This needs a bit more explanation

Reply:
https://github.com/ably/ably-cocoa/pull/1491#discussion_r962945136